ioSPI/atomic_models.py

```diff
@@ -148,7 +148,7 @@ def clean_gemmi_structure(structure=None):
     return structure
 
 
-def write_gemmi_model(path, model=gemmi.Model("model")):
+def write_atomic_model_to_pdb(path, model=gemmi.Model("model")):
```

Comment:
This can also be writing the atomic_model to mmCIF: I'd say:
- either write_atomic_model_to_file
- or write_atomic_model_to_pdb_or_mmcif (2)
- or write_atomic_model (1)

My preference would be either (1) or (2): "file" is not informative, (it's like "utils", it does not bring anything) thus it is not worth a user's time to have to type it in.

Which ever you choose, the same convention should be used for read_atomic_model_from_*** above.